frameworks/template/README.md

```diff
@@ -396,58 +387,50 @@ In case of problems with accessing the DC/OS Dashboard, logs may also be accesse
 1. You should now see two lists of tasks. `Active Tasks` are what's currently running, and `Completed Tasks` are what has since exited. Click on the `Sandbox` link for the task you wish to examine.
 1. The `Sandbox` view will list files named `stdout` and `stderr`. Click the file names to view the files in the browser, or click `Download` to download them to your system for local examination. Note that very old tasks will have their Sandbox automatically deleted to limit disk space usage.
 
-<a name="knownissues"></a>
-# Known Issues
-
-- _LIST ANY KNOWN BUGS OR ISSUES WITH YOUR SERVICE INTEGRATION (AND THEIR WORKAROUNDS) HERE_
 
 <a name="limitations"></a>
 # Limitations
 
-- _LIST SUMMARY OF CAVEATS OR USEFUL KNOWLEDGE FOR RUNNING THE SERVICE HERE_
-- Shrinking cluster size (number of nodes) is not supported.
-
-## Removing a Node
-
-Removing a node is not supported at this time.
-
-## Automatic Failed Node Recovery
-
-Nodes are not automatically replaced by the service in the event a system goes down. You may either manually replace nodes as described under [Managing](#managing), or build your own ruleset and automation to perform this operation automatically.
+_MANAGE CUSTOMER EXPECTIONS BY DISCLOSING ANY FEATURES OF YOUR PRODUCT THAT ARE NOT SUPPORTED WITH DC/OS, FEATURES MISSING FROM THE DC/OS INTEGRATION, ETC._
 
 ## Updating Storage Volumes
 
 Neither volume type nor volume size requirements may be changed after initial deployment.
 
 ## Rack-aware Replication
 
-Rack awareness within the service is not currently supported, but is planned to be supported with a future release of DC/OS.
+Rack awareness placement and awareness is not support at this time.
```

Comment:
The use of the word "awareness" twice seems awkward; recommend removing the first instance.